### PR TITLE
Update ide.md

### PR DESCRIPTION
**Pull request by dbadia**
Update openhab component selection steps for current eclipse installer version

### CODE REVIEW

File: developers/development/ide.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The Eclipse IDE is used for openHAB developments. The Eclipse Installer automati
 ![Step 0](images/ide0.png)
 3. Choose the "Eclipse IDE for Java Developers" and select "Next":
 ![Step 1](images/ide1.png)
-4. Expand the "Github.com/openHAB" entry, double click "openHAB Development" (the double click is important: The entry has to appear in the empty table at the bottom). Furthermore double-click all entries that you would want to have available in your workspace (you can select multiple/all of them). Make sure that all of them are listed in the table at the bottom and select "Next".
+4. Expend "Github Projects", then "openHAB".  Select the "openHAB Development" item and any other entries that you want to have available in your workspace (you can select multiple/all of them).  Click "Next" when you are finished.
```
Comment (Confectrician) [inline developers/development/ide.md]: "Expend" -> "Expand"

Comment (Confectrician) [inline developers/development/ide.md]: `"openHAB".  Select` there one whitespace more than need before "Select"

Comment (dbadia) [reply developers/development/ide.md]: Glad you caught that

Comment (dbadia) [reply developers/development/ide.md]: Updated

**Comment by dbadia** [comment]
Signed-off-by: Joe Smith joe.smith@email.com (github: github_handle)

**Comment by dbadia** [comment]
Hi @Confectrician 
Ha, sorry, I completely misunderstood.  I have updated the sign-off comment with my info

Thanks for your help on this!
Dave

**Pull request merged**